### PR TITLE
Tighten up text and update links

### PR DESCRIPTION
**Pull request by bermudezmt**
Hey @justinvp - thought I'd fix some of the issues I noticed on this shell script that you've referenced in https://github.com/pulumi/docs/issues/1176#issuecomment-509876249.

### CODE REVIEW

File: dist/install.sh [hunk 1]
```diff
@@ -177,7 +177,7 @@ say_blue "=== Pulumi is now installed! 🍹 ==="
 if [ "$EXTRA_INSTALL_STEP" != "" ]; then
     say_white "${EXTRA_INSTALL_STEP}"
 fi
-say_green "+ If you're new to Pulumi, here are some resources for getting started:"
-say_green "      - Getting Started Guide: https://pulumi.io/quickstart"
+say_green "+ If you're new to Pulumi, here are some resources to get you started:"
+say_green "      - Getting Started Guide: https://www.pulumi.com/docs/quickstart"
 say_green "      - Examples Repo: https://github.com/pulumi/examples"
 say_green "      - Create a New Project: Run 'pulumi new' to create a new project using a template"
```
Comment (bermudezmt) [inline dist/install.sh]: I'm not sure "Create a New Project" is descriptive enough,  but wasn't sure we also wanted this section to be very verbose. What do we mean exactly by "using a template" in this case?

File: dist/install.sh [hunk 2]
```diff
@@ -45,8 +45,8 @@ at_exit()
 {
     if [ "$?" -ne 0 ]; then
         >&2 say_red
-        >&2 say_red "We're sorry, but it looks like something might have gone wrong during install."
-        >&2 say_red "If you need help, please join the #install channel on https://slack.pulumi.io/"
+        >&2 say_red "We're sorry, but it looks like something might have gone wrong during installation."
+        >&2 say_red "If you need help, please join the #install channel on https://pulumi-community.slack.com/"
```
Comment (ericrudder) [inline dist/install.sh]: "If you need help, please join us on https://pulumi-community.slack.com/"

(our channels change from time to time)

Comment (bermudezmt) [reply dist/install.sh]: I changed it. :)

File: dist/install.sh [hunk 3]
```diff
@@ -10,8 +10,8 @@ WHITE="\\033[37;1m"
 
 print_unsupported_platform()
 {
-    >&2 say_red "error: We're sorry to say that it looks like Pulumi is not supported on your platform"
-    >&2 say_red "       we support 64 bit versions of Linux and macOS but we're interested in supporting"
+    >&2 say_red "error: We're sorry,but it looks like Pulumi is not supported on your platform"
```
Comment (justinvp) [inline dist/install.sh]:
```suggestion
    >&2 say_red "error: We're sorry, but it looks like Pulumi is not supported on your platform"
```

File: dist/install.sh [hunk 4]
```diff
@@ -56,7 +56,7 @@ VERSION=""
 if [ "$1" = "--version" ] && [ "$2" != "latest" ]; then
     VERSION=$2
 else
-    if ! VERSION=$(curl --fail --silent -L "https://pulumi.io/latest-version"); then
+    if ! VERSION=$(curl --fail --silent -L "https://www.pulumi.com/latest-version"); then
```
Comment (justinvp) [inline dist/install.sh]: I already have a PR open that changes this #43

File: dist/install.sh [hunk 5]
```diff
@@ -177,7 +177,7 @@ say_blue "=== Pulumi is now installed! 🍹 ==="
 if [ "$EXTRA_INSTALL_STEP" != "" ]; then
     say_white "${EXTRA_INSTALL_STEP}"
 fi
-say_green "+ If you're new to Pulumi, here are some resources for getting started:"
-say_green "      - Getting Started Guide: https://pulumi.io/quickstart"
+say_green "+ If you're new to Pulumi, here are some resources to get you started:"
+say_green "      - Getting Started Guide: https://www.pulumi.com/docs/quickstart"
```
Comment (justinvp) [inline dist/install.sh]: Please revert these changes. Per #38, we want to have a single call-to-action here, which is already taken care of in #43. Happy for any feedback on the wording in #43.

File: dist/install.sh (hunk 1 above)
Comment (justinvp) [reply dist/install.sh]: This line will be deleted with #43.

File: dist/install.sh (hunk 5 above)
Comment (bermudezmt) [reply dist/install.sh]: I only reverted lines 180-181 0 should I do the same for line 59?

**Comment by chrsmith** [comment]
Going to look into why Travis if failing and when green will check this in, since it contains the fix for part of https://github.com/pulumi/docs/issues/1311 (updating references to pulumi.io to be pulumi.com).

File: dist/install.sh [hunk 6]
```diff
@@ -45,8 +45,8 @@ at_exit()
 {
     if [ "$?" -ne 0 ]; then
         >&2 say_red
-        >&2 say_red "We're sorry, but it looks like something might have gone wrong during install."
-        >&2 say_red "If you need help, please join the #install channel on https://slack.pulumi.io/"
+        >&2 say_red "We're sorry, but it looks like something might have gone wrong during installation."
+        >&2 say_red "If you need help, please join us on https://pulumi-community.slack.com/"
```
Comment (justinvp) [inline dist/install.sh]: @chrsmith, should we be linking to https://pulumi-community.slack.com/ or https://slack.pulumi.com/?

Comment (chrsmith) [reply dist/install.sh]: Good catch, yes. We should always send folk to slack.pulumi.com, since that way they can get an invite. If they don't have access to the slack organization already it won't work for them.

I'll make the fix.

**Comment by justinvp** [comment]
> Going to look into why Travis if failing and when green will check this in

Probably failing because this PR is from a fork.

**Comment by chrsmith** [comment]
Yes, Travis is failing because its a fork. But somehow I was able to make a commit to it, even though Maria never granted me write access? Anyways, I'll use admin super powers to merge without Travis being green and just watch the `master` push branch to confirm it gets deployed successfully.
